### ISSUE TITLE
Multi-line DATABASE_INFO

### ISSUE THREAD

**Issue by richardotis** [pycalphad/pycalphad#228]
Certain valid TDB expressions give the parser trouble. One example is the multi-line `DATABASE_INFO` command, which pycalphad should silently ignore, but instead raises an error:

```python
from pycalphad import Database
Database('''
DATABASE_INFO'
'
'
!
''')
```
```
Failed while parsing: DATABASE_INFO' ' ' 
Tokens: None
---------------------------------------------------------------------------
ParseException                            Traceback (most recent call last)
[snip]
```

**Comment by bocklund** [comment]
This error is actually caused by the `'` at the end of `DATABASE_INFO'`. If this is changed to be `DATABASE_INFO '` (with a space between the tokens `DATABASE_INFO` and `'`), it works.

```python
# fails
from pycalphad import Database
Database('''
DATABASE_INFO'
'
'
!
''')

# works
from pycalphad import Database
Database('''
DATABASE_INFO '
'
'
!
''')
```

@richardotis Is the quote without a space a common TDB idiom (i.e. do we need to change the TCCommand instantiation to handle this common case) or should we properly treat this as an error? FWIW, none of the databases I have list the quotation directly after the DATABASE_INFO, there's always a space.

In addition, the Thermo-Calc database manager user guide actually lists this command as

```
DATABASE_INFORMATION [text]*10000 !
```

implying that a space should be there (and the command should be `INFORMATION`, not `INFO`, which we currently have.

**Comment by richardotis** [comment]
You are correct about the true cause of the error (which I realized awhile ago but neglected to document here). I have no idea on prevalence, but the above snippet comes from a TDB I found in the wild, so I assume Thermo-Calc will accept the syntax.

The database manager guide says that `'` is a newline character for the purposes of rendering the text (which pycalphad currently doesn't do). My reading is that `DATABASE_INFO'` is a syntax error, but if TC accepts it, probably we should too; by implicitly adding a space, or parsing as if there were a space after the command and before the quote. I can't think of anywhere else in a TDB file I would expect to run into this issue, but probably it should be a modification of TCCommand and not just a special case.

The `DATABASE_INFORMATION` command is improperly shortened in the current TDB parser, and fixing it would be a great first bug.

**Comment by bocklund** [comment]
Needs a test to see if commercial software reads a database containing `DATABASE_INFO'` without an error
